apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.IntegrationTests/HighPerformancePubSubTest.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright 2017, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.PubSub.V1.IntegrationTests
+{
+    // Notes:
+    // * These tests output progress to the console, as they can take minutes to run.
+    // * These send/recv considerable amounts of network data.
+    //   Only test with a machine and cloud account that can handle this.
```

Comment:
Hmm. Given the warning, I wonder whether we need to create a new kind of test project - StressTests or similar. (Ben has the same requirement for Spanner.) I normally run all the integration tests before doing a release, and we probably don't want to do that from my home machine/network/account... it's probably okay for now, but let's think about it.